ios/OpenTokReactNative/OTSessionManager.swift

```diff
@@ -205,91 +205,90 @@ class OTSessionManager: RCTEventEmitter {
             OTRN.sharedState.sessionDisconnectCallbacks[sessionId] = callback;
         }
     }
-    
+
     @objc func publishAudio(_ publisherId: String, pubAudio: Bool) -> Void {
         guard let publisher = OTRN.sharedState.publishers[publisherId] else { return }
         publisher.publishAudio = pubAudio;
     }
-    
+
     @objc func publishVideo(_ publisherId: String, pubVideo: Bool) -> Void {
         guard let publisher = OTRN.sharedState.publishers[publisherId] else { return }
         publisher.publishVideo = pubVideo;
     }
-    
+
     @objc func subscribeToAudio(_ streamId: String, subAudio: Bool) -> Void {
         guard let subscriber = OTRN.sharedState.subscribers[streamId] else { return }
         subscriber.subscribeToAudio = subAudio;
     }
-    
+
     @objc func subscribeToVideo(_ streamId: String, subVideo: Bool) -> Void {
         guard let subscriber = OTRN.sharedState.subscribers[streamId] else { return }
         subscriber.subscribeToVideo = subVideo;
     }
-    
+
     @objc func setPreferredResolution(_ streamId: String, resolution: NSDictionary) -> Void {
         guard let subscriber = OTRN.sharedState.subscribers[streamId] else { return }
         subscriber.preferredResolution = Utils.sanitizePreferredResolution(resolution);
     }
-    
+
     @objc func setPreferredFrameRate(_ streamId: String, frameRate: Float) -> Void {
         guard let subscriber = OTRN.sharedState.subscribers[streamId] else { return }
         subscriber.preferredFrameRate = Utils.sanitizePreferredFrameRate(frameRate);
     }
-    
+
     @objc func changeCameraPosition(_ publisherId: String, cameraPosition: String) -> Void {
         guard let publisher = OTRN.sharedState.publishers[publisherId] else { return }
         let position = cameraPosition == "front" ? AVCaptureDevice.Position.front: AVCaptureDevice.Position.back
-        
+
+        publisher.cameraPosition = position
```

Comment:
needed to pass the current camera position to the publisher, so the opentok sdk will enable or disable the mirror effect